procedures/MT/rotator/rotator_analysis.ipynb

```diff
@@ -0,0 +1,914 @@
+{
```

Comment:
<p>Consider:</p><p>Declaration of User-defined Functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done in the commit: [3f0a032](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/3f0a032629e2db3344107ff9821c35efe5e44afd). Thanks!